app/qml/ActiveLayerPanel.qml

```diff
@@ -14,7 +14,6 @@ import QgsQuick 0.1 as QgsQuick
 import "."  // import InputStyle singleton
 
 Drawer {
-    property string title: qsTr("Survey Layer")
```

Comment:
@PeterPetrik - you mentioned to remove this - is this what you meant?

Reply:
yes, it is not used anywhere